geoips/plugins/yaml/products/ami.yaml

```diff
@@ -62,3 +62,34 @@ spec:
             name: matplotlib_linear_norm
             arguments:
               cbar_label: "Reflectance, AMI B03 0.64um Red Band"
+    - name: absdiff-IR-BD
```

Comment:
Does this actually have anything to do with IR-BD?  This is just plotting the difference between two scan times for the 11.2 um channel?

Reply:
I was using the same variable as IR-BD was so that's why I called it that. In reality it's just an absolute difference between two scan times for the channel you mentioned. If we want to name it something else I'm fine with that.

Reply:
I don't think it's worth changing it for this PR, feel free to resolve this if you don't care either!

Reply:
Resolving!